presentation/src/main/java/me/androidbox/presentation/event/viewmodel/EventViewModel.kt

```diff
@@ -32,12 +34,19 @@ class EventViewModel @Inject constructor(
     private val verifyVisitorEmailUseCase: VerifyVisitorEmailUseCase,
     private val preferenceRepository: PreferenceRepository,
     private val alarmScheduler: AlarmScheduler,
-    private val uploadEvent: UploadEvent
+    private val uploadEvent: UploadEvent,
+    savedStateHandle: SavedStateHandle
 ) : ViewModel() {
 
     private val _eventScreenState: MutableStateFlow<EventScreenState> = MutableStateFlow(EventScreenState())
     val eventScreenState = _eventScreenState.asStateFlow()
 
+    init {
+        savedStateHandle.get<String>("event_id")?.let { eventId ->
+            eventRepository.getEventById(eventId)
```

Comment:
You should set the event's screen state information with this event you get from the repository

Reply:
@MKiperszmid Updated this already. I am not calling a function that collect this state from the DB